config/gameConfig.json

```diff
@@ -271,6 +271,19 @@
 					"portraitYoungYog" : 162
 				}
 			},
+			"chronicles" : { 
+				"supported" : true,
+				"iconIndex" : 2,
```

Comment:
No icon for this format?

Reply:
There are basically separate icons. But a different for each chronicle. So our current logic don't work yet.

And there shouldn't be maps with this format out there. Because of that the only way to see this icon is on loading/saving screen. Showing SoD icon is ok for now, I think.